### ISSUE TITLE
BaseResponse:: getWebhookUrl() doesn't handle new webhooks types properly

### ISSUE THREAD

**Issue by ghimes** [mailgun/mailgun-php#740]
If you create new webhooks the API response looks like:
```php
[webhook:Mailgun\Model\Webhook\BaseResponse:private] => Array
        (
            [urls] => Array
                (
                    [0] => http://domain/index.php/
                )

        )
```

but the method that should fetch the webhook url looks like:
```php
public function getWebhookUrl(): ?string
    {
        return $this->webhook['url'] ?? null;
    }
```
It looks for a `url` key and is returning a string, while the response actually contains an `urls` key which holds an array. 

The way it is now is working only for legacy webhooks, but not for the new webhooks like: `complained` and `permanent_fail`

**Comment by DavidGarciaCat** [comment]
Potentially related to PR https://github.com/mailgun/mailgun-php/pull/609 that still needs a final review

**Comment by DavidGarciaCat** [comment]
This should be fixed as part of release 3.5.0.

Please check the changelog notes and the new methods to handle both a single URL or an array of URLs.
